addons/openebs/template/generate.sh

```diff
@@ -23,6 +23,11 @@ function generate() {
         --set defaultStorageConfig.enabled=false \
         --set localprovisioner.enableDeviceClass=false \
         --set localprovisioner.enableHostpathClass=false \
+        --set release.version="$chart_version" \
```

Comment:
this hardcodes imageTags from README.md
https://github.com/openebs/charts/blob/openebs-3.3.0/charts/openebs/README.md?plain=1#L90

Reply:
do we want to hardcode images, though?
won't this prevent us from getting a new ndm image if one comes out in the future?

we're not regenerating old versions, so I don't see the problem with letting them float

Reply:
I am working around an upstream issue I created. They forgot to update the images for the new version of the chart.

https://github.com/openebs/charts/issues/309